autosklearn/pipeline/base.py

```diff
@@ -34,6 +34,7 @@ class BasePipeline(Pipeline):
 
     def __init__(
         self,
+        feat_type=None,
         config=None,
```

Comment:
Add types here since there no types for the other attributes ?

Reply:
Yes, please. We need to somehow start :)